Sage/carbon
# Icon Font Updated

# Description

Updated Icon Font to be aligned further with DLS and added all required Icons that were still in SVG format into the Icon Font itself and remove all SVGs for Icons.

# Visuals

All Icons in the Icon Font now:

![Screenshot 2019-09-05 at 15 31 54](https://user-images.githubusercontent.com/2411024/64351497-52d16280-cff2-11e9-90ed-f7935eaefffb.png)

# Testing Instructions

Ensure using the Icons that were SVGs still work using the Icon Font.

Icons to check in particular:

- in_transit
- arrow_left_small
- arrow_right_small
- progress
- bin
- basket
- credit_card_slash
- plus_large
- minus_large
- remove
- grid
- location
- coins
